web-jehlomat/src/types.ts

```diff
@@ -19,3 +19,24 @@ export interface IUser {
     isAdmin: boolean;
 }
 
+type Location = {
+    id: number;
+    mestkaCast: string | null;
+    mestkaCastName: string | null;
+    obec: string | null;
+    obecName: string | null;
+    okres: string | null;
+    okresName: string | null;
+};
+export interface ITeam {
```

Comment:
Just out of curiosity, why do you prefix interfaces with `I`?

Reply:
it is just a convention I am used to, so using it automatically when typing

Reply:
I see the intention and as a C# developer I'm used to use it in C#, however not in TypeScript. But I guess it's because I've worked with Angular and also Vue in TypeScript and they doesn't use `I` prefix in ecosystem, similarly like they don't use postfix `Impl` for classes, which comes from Java.
But I have to admit, the naming is always hard :)